_posts/en/newsletters/2019-03-19-newsletter.md

```diff
@@ -0,0 +1,277 @@
+corresponding `OP_0` to `OP_16` opcode, and the commitment may be a
+different length (up to 40 bytes), but nothing else changes.
+
+What you see above is the entire fundamental change you need to make on
+the backend of your software in order to enable sending to bech32
+addresses.  For most platforms, it should be a very easy change.  See
+[BIP173][] and the [reference implementations][bech32 ref libs] for a
+set of test vectors you can use to ensure your implementation works
+correctly.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], and [Bitcoin Improvement Proposals
+(BIPs)][bips repo].*
+
+- [LND #2022][] allows the creation of "hold invoices".  These are standard
+  LN invoices that are processed differently when payment is received.
+  Instead of the receiver immediately returning the payment preimage
+  upon receipt of the payment, the receiver delays up until the maximum
+  allowed by the payment timelock.  This allows the receiver to accept
+  or reject the payment subsequent to knowing that the money is
+  available.  For example, Alice could automatically generate hold
+  invoices on her website but wait until a customer actually paid before
+  searching her inventory for the requested item.  This would give her a
+  chance to cancel the payment if she couldn't deliver.  Other example
+  use cases are provided in the PR's main description.
+
+- [LND #2618][] implements most of the code necessary for an initial
+  version of watchtower client support that will allow an LND node to
+  pair with a private watchtower and send it encrypted state backups.
+  The watchtower can then monitor the block chain for attempt channel
+  contract breaches and submit breach remedy (justice) transactions that
+  prevent the honest party from losing funds.  See the notable code
+  changes sections of our previous newsletters for coverage of the
+  commits implementing the server-side watchtower changes:
+  [#7][newsletter #7], [#19][newsletter #19], [#22][newsletter #22], and
+  [#30][newsletter #30].
+
+- [C-Lightning #2342][] adds a new `setchannelfee` RPC that allows the
+  user to set the feerates for each of their channels individually.
+
+{% include references.md %}
+{% include linkers/issues.md issues="2022,2618,2342" %}
+[esplora convo]: http://www.erisian.com.au/bitcoin-core-dev/log-2019-03-08.html#l-53
+[havar github]: https://github.com/Blockstream/esplora/issues/51
+[mit vids]: https://www.youtube.com/user/MITBitcoinClub/videos
+[0.18.0]: https://bitcoincore.org/bin/bitcoin-core-0.18.0/
+[noinput thread]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-March/016766.html
+[ivgi twitter]: https://twitter.com/shesek/status/1103320174936109057
+[uih2]: https://gist.github.com/AdamISZ/4551b947789d3216bacfcb7af25e029e#gistcomment-2796539
```

Comment:
I prefer that gist comment because I can link directly to it rather than requiring people read through waxwing's post about an unrelated thing.  To solve the ephemeral problem, I [archived it](https://archive.fo/kimMH).